tests/driver_sam0_eth/Kconfig

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2022 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config APPLICATION
+    bool
+    default y
+    depends on HAS_CPU_SAMD5X
+    depends on HAS_PERIPH_ETH # TODO: complete with other SAM0 CPUs that have ethernet
```

Comment:
@benpicco has a good overview of the sam0 series.  Are there any more boards/cpus that may have the ethernet peripheral?

Reply:
The whole SAM E5x line has Ethernet (it's SAM D5x with **e**thernet like the SAM R21 is SAM D21 with **r**adio).
But we only have the `same54-xpro` board in RIOT with a CPU from the line.